frontend/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
Can you add this line:

```json
"type": "module",
```

I believe this is required in order to use the standard ECMAScript module syntax (`export blah`, `import blah from "blah"`) in normal `.js` files. Otherwise, you have to use the older CommonJS syntax (`module.exports = blah`, `const blah = require("blah")`) or use the `.mjs` extension on all JS files.

See https://nodejs.org/api/esm.html#introduction, https://nodejs.org/api/packages.html#type. Also consider doing this in `backend/package.json` for consistency, although I think it's pretty common for backend code/online examples to use CommonJS syntax so I'll leave it up to you

Reply:
Interesting, I was looking into this earlier but I was under the impression it was reversed for some reason (i.e. module.exports/require needed module). Added it to both.